Content.Shared/Access/Components/IdCardConsoleComponent.cs

```diff
@@ -3,8 +3,6 @@
 using Robust.Shared.GameStates;
 using Robust.Shared.Prototypes;
 using Robust.Shared.Serialization;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.List;
-using Robust.Shared.Prototypes;
```

Comment:
Dont remove this even if its unused if its from upsteam.

Reply:
this must have been done by re-sharper

Reply:
resolved